src/pocketmine/plugin/PluginDescription.php

```diff
@@ -75,6 +76,17 @@ private function loadMap(array $plugin){
 		if(isset($plugin["depend"])){
 			$this->depend = (array) $plugin["depend"];
 		}
+		if(isset($plugin["extensions"])){
+			$extensions = (array) $plugin["extensions"];
+			$isLinear = $extensions === array_values($extensions);
+			foreach($extensions as $k => $v){
+				if($isLinear){
+					$k = $v;
+					$v = "*";
+				}
+				$this->extensions[$k] = is_array($v) ? $v : [$v];
```

Comment:
hand stuck in toaster...

can we assume the type of `extensions` to be `string[][]`?